lib/widgets/message_list.dart

```diff
@@ -433,11 +433,15 @@ class MarkAsReadWidget extends StatelessWidget {
             style: FilledButton.styleFrom(
               backgroundColor: _UnreadMarker.color,
               minimumSize: const Size.fromHeight(38),
-              textStyle: const TextStyle(
-                fontFamily: 'Source Sans 3',
-                fontSize: 18,
-                height: (23 / 18),
-              ).merge(weightVariableTextStyle(context)),
+              textStyle:
+                // Restate [FilledButton]'s default, which inherits from
+                // [zulipTypography]…
+                Theme.of(context).textTheme.labelLarge!
+                // …then clobber some attributes to follow Figma:
+                .merge(const TextStyle(
+                  fontSize: 18,
+                  height: (23 / 18))
+                .merge(weightVariableTextStyle(context, wght: 400))),
```

Comment:
Hmm interesting. I guess this will be a case to consider when thinking about how we ultimately want to arrange the data flow for the text styles.